.nuxt-guide/README.md

```diff
@@ -278,14 +291,120 @@ Then, wrap the entire template with `<div>` and replace `<LayoutGuest>` with `<N
 </template>
 ```
 
-## Replace `<RouterLink>` with `<NuxtLink>`
+## Copy other pages
+
+Copy the pages from `src/views/` (except `LoginView.vue` and `HomeView.vue`) to to `pages/` and remove the `View` from the file names
+
+Now for each page wrap the entire template with `<div>` and replace `<LayoutAuthenticated>` with `<NuxtLayout>` with a `name` prop just like you did for `dashboard.vue` before.
+
+Note: For the `Error.vue` and `Style.vue` pages, use the `<NuxtLayout>` without the  `name` prop.
+
+## Update `configs/menuAside.js`
+
+Update `Styles` and `Login` links:
+```js
+// ...
+  {
+    // to: "/",
+    to: "/style",
+    label: "Styles",
+    icon: mdiPalette,
+  },
+  // ...
+  {
+    to: "/login",
+    to: "/",
+    label: "Login",
+    icon: mdiLock,
+  },
+// ...
+```
+
+## Replace `RouterLink` with `NuxtLink`
+
+Search all files for `RouterLink` and replace import statements and references to `RouterLink` as per below example:
+
+```vue
+<script setup>
+// import { RouterLink } from 'vue-router'
+import { NuxtLink } from '#components'
+
+// ...
 
-Details are coming soon...
+const componentClass = computed(() => {
+  if (props.item.to) {
+    // return RouterLink
+    return NuxtLink
+  }
+})
```

Comment:
Hi, I don't understand this, can you give a proper example?

because if I search for the text `RouterLink` in all files, there is already a `cons componentClass`

example: https://github.com/justboil/admin-one-vue-tailwind/blob/99bfa7b3b217da5406ca674c5edae4561473c2e3/src/components/AsideMenuItem.vue#L27-L32